pages/linux/parted.md

```diff
@@ -0,0 +1,36 @@
+# parted
+
+> A partition manipulation program.
+> More information: <https://www.gnu.org/software/parted/parted.html>.
+
+- List partitions on all block devices:
+
+`parted --list`
+
+- Start to manipulate disk partition:
+
+`parted {{/dev/sdX}}`
+
+- Create a new partition table of label-type directly, label-type can be gpt, msdos etc:
+
+`parted --script {{/dev/sdX}} mklabel {{gpt}}`
```

Comment:
```suggestion
`sudo parted --script {{/dev/sdX}} mklabel {{gpt}}`
```

Reply:
Great catch, @aaaawwWWWwwwwWWW! Thanks :D

(psst, once you get 5 non-trivial PRs merged, you're eligible to become a contributor with the power to review & merge PRs according to our [community roles document](https://github.com/tldr-pages/tldr/blob/master/COMMUNITY-ROLES.md) :D)